apps/api/src/app.module.ts

```diff
@@ -28,7 +29,24 @@ const coreModules = [
 ];
 
 @Module({
-  imports: [...coreModules, ...apiModules],
+  imports: [
+    ...coreModules,
+    ...apiModules,
+    AuthModule,
+    AuthModule.forRoot({
+      // These are the connection details of the app you created on supertokens.com
+      connectionURI: process.env.SUPERTOKENS_URI as string,
+      apiKey: process.env.SUPERTOKENS_API_KEY as string,
+      appInfo: {
+        // Learn more about this on https://supertokens.com/docs/passwordless/appinfo
+        appName: process.env.APP_NAME as string,
```

Comment:
Please stop using process env. To use this in nest js we have config module already used. Kindly refer the doc

https://docs.nestjs.com/techniques/configuration